catalog/acm/README.md

```diff
@@ -0,0 +1,90 @@
+# GKE ACM blueprint
+
+A GKEHub Membership and ACM enrollment blueprint for Config Sync and Policy Controller for a pre-provisioned GKE cluster
+
+## Setters
+
+```
+Setter                    Usages
+auditIntervalSeconds      2
+cluster-name              5
```

Comment:
We should use kebab-case for setters

Reply:
OK, I will change these.

Reply:
changed variable names to use kebab-case